README.md

```diff
@@ -1 +1,58 @@
-# Test-REDDS
\ No newline at end of file
+# Project REDSS
```

Comment:
This doesn't match the project title on gh

Reply:
Thanks @as2388 resolved in the new branch https://github.com/AfricasVoices/Test-REDDS/tree/coding_schemes.